project/GrasshopperBuild.scala

```diff
@@ -18,7 +18,7 @@ object BuildSettings {
       organization  := buildOrganization,
       version       := buildVersion,
       scalaVersion  := buildScalaVersion,
-      wartremoverWarnings ++= Warts.allBut(Wart.NoNeedForMonad, Wart.NonUnitStatements),
+      //wartremoverWarnings ++= Warts.allBut(Wart.NoNeedForMonad, Wart.NonUnitStatements),
```

Comment:
Removing the wart remover doesn't actually remove the warts. :)  Did you mean to leave this commented out?